napari/utils/colormaps/colormap_utils.py

```diff
@@ -118,12 +118,22 @@ def _all_rgb():
     return np.stack((r, g, b), axis=-1).reshape((-1, 3))
 
 
-# obtained with colorconv.rgb2luv(_all_rgb().reshape((-1, 256, 3)))
+# The following values were precomputed and stored as constants
```

Comment:
Is it possible to do this at install time? Not that I think that is necessary in this case...

Reply:
I'm not aware of a way to do that in Python, but that doesn't mean it doesn't exist!